README.md

```diff
@@ -96,7 +96,7 @@ Add the repo to your project inside your `pom.xml` file
 ```
 
 ##### Gradle
-Add the repo to your project by adding `mavenLocal()` to `repositories` and `implementation 'com.sipios:spring-search:0.2.0'` to  `dependencies` in your `build.gradle` file.
```

Comment:
Why did you remove this instruction?

Reply:
I did not make this change, it’s a previous commit which has been done to master, I made changes to the `feature/add-more-types` and merged it with master in my local repo.